mage_ai/server/server.py

```diff
@@ -229,9 +235,21 @@ async def main(
                     email='admin@admin.com',
                     password_hash=create_bcrypt_hash('admin', password_salt),
                     password_salt=password_salt,
-                    owner=True,
+                    roles_new=[Role.query.filter(Role.name == 'owner').first()],
                     username='admin',
                 )
+        else:
+            if not user.roles_new:
+                all_users = User.query.all()
+                for user in all_users:
```

Comment:
In the "if branch", I would suggest using a different variable name than "user". Or you can rename the user above to "owner_user".